draft-davidben-tls-trust-expr.md

```diff
@@ -770,7 +770,7 @@ The negotiation mechanism described in this document is analogous to the `certif
 
 Thus, this mechanism SHOULD NOT be used to advertise trust anchors or distrusts which are unique to an individual user. Rather, trust expressions SHOULD be computed based only on the trust anchors common to the relying party's anonymity set ({{Section 3.3 of !RFC6973}}). Additionally, multiple trust expressions may evaluate to the same trust anchor list, so relying parties in the same anonymity set SHOULD send the same trust expression. To achieve this, trust expressions SHOULD be assembled by the root program and configured in relying parties alongside trust store updates.
 
-For example, a web browser may support both a common set of trust anchors configured by the browser vendor, along with user-specified additions and removals. The common trust anchors would reveal, at most, which browser is used, while the user-specified modifications may reveal identifying information about the user. The trust expression SHOULD reflect only the common trust anchors. This limits the benefits of trust anchor agility in two ways:
+For example, a web browser may support both a common set of trust anchors configured by the browser vendor, along with user-specified additions and removals. The common trust anchors would reveal, at most, which version of which browser is used, while the user-specified modifications may reveal identifying information about the user. The trust expression SHOULD reflect only the common trust anchors. This limits the benefits of trust anchor agility in two ways:
```

Comment:
I think the discussion at the IETF thread has slightly misrepresented the point of this paragraph. It is not a complete analysis of all web browser scenarios. Rather, it is an illustrative example in service of the point of a Privacy Considerations section: to guide implementors in using the mechanism according to privacy expectations. Given that confusion, I think it's worth rephrasing this a bit to be more clear about the scope. (Trying to do a complete analysis will be very product-specific, and depend on things like frequency of trust store updates, which browsers share root programs, and whether the browser has an out-of-band update process.)

Maybe something like this?

```suggestion
For example, a web browser may support both a common set of trust anchors configured by the browser vendor, along with user-specified additions and removals. Applying the above guidance with a typical browser's desired anonymity set, the trust expression would reflect only the common trust anchors. User-specified modifications may reveal identifying information about the user.

This guidance limits the benefits of trust anchor agility in two ways:
```

I tried to say something about what common trust anchors reveal, but it got really, really long-winded and awkward to cover all bases. WDYT?

Reply:
I tweaked your text a bit. have a look